rules/no-invalid-remove-event-listener.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+const {methodCallSelector, matches} = require('./selectors');
+
+const MESSAGE_ID = 'no-invalid-remove-event-listener';
+const messages = {
+	[MESSAGE_ID]: 'The listener argument should be a function reference.'
+};
+
+const isBindCall = node =>
+	node.type === 'CallExpression' &&
+	node.callee.type === 'MemberExpression' &&
+	node.callee.property.name === 'bind' &&
+	!node.callee.computed;
```

Comment:
This function isn't used anymore. Do the below matchers handle when the callee is computed?

Reply:
Yes, remove it, all tested.